static/css/page-user.less

```diff
@@ -152,13 +152,19 @@ th.toggle-all {
     }
   }
 }
-
+/* stylelint-disable no-descending-specificity */
```

Comment:
... if you use BEM, the idea here we can remove this line...

Reply:
Even after making the changes, I get this error.
```
Expected selector ".olform--decoration" to come before selector ".import-export    no-descending-specificity
           .olform--decoration"
```

Reply:
I think we are using `.olform--decoration` only for import/export so I guess we can just remove the `width: 350px;`